test/performance/alphabet/alphabet_assign_rank_benchmark.cpp

```diff
@@ -53,4 +60,32 @@ BENCHMARK_TEMPLATE(assign_rank_, seqan3::masked<seqan3::dna4>);
 BENCHMARK_TEMPLATE(assign_rank_, seqan3::qualified<seqan3::dna4, seqan3::phred42>);
 BENCHMARK_TEMPLATE(assign_rank_, seqan3::qualified<seqan3::dna5, seqan3::phred63>);
 
+#if SEQAN3_HAS_SEQAN2
+template <typename alphabet_t>
+void assign_rank_2(benchmark::State & state)
+{
+    using rank_t = uint8_t;
+
+    std::array<rank_t, 256> ranks{};
+    uint8_t i = 0;
+    for (rank_t & r : ranks)
+        r = i++ % seqan::ValueSize<alphabet_t>::VALUE;
```

Comment:
```suggestion
    std::array<rank_t, 256> ranks{};
    fill_rank_array(ranks, seqan::ValueSize<alphabet_t>::VALUE);
```
And then use this also in the upper benchmark for SeqAn3:

```cpp
template <typename rank_array_t>
void fill_rank_array(rank_array_t & ranks, size_t const alphabet_size)
{
    uint8_t i = 0;
    for (rank_t & r : ranks)
        r = i++ % alphabet_size;
}
```